controllers/auth.js

```diff
@@ -0,0 +1,104 @@
+const jwt = require('jsonwebtoken');
+const mongoose = require('mongoose');
+const utils = require('./utils');
+const http = require('http');
+const express = require('express');
+const request = require('request');
+
+// if(process.env.STATUS == "development"){
```

Comment:
Can we implement this or remove it?

Reply:
Implemented separate STATUS check in utils. Removed from routes for cleanliness.